screens/ImageScreen.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  Button,
+  Clipboard,
+  Image,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+import * as firebase from 'firebase';
+import * as Permissions from 'expo-permissions';
+import KitText from '../components/KitText';
+import KitButton  from '../components/KitButton'
+import Colors from '../constants/Colors';
+import FontStyles from '../constants/FontStyles';
+import uuid from 'uuid';
+
+export default class ImageScreen extends React.Component{
+  state = {
+    image: null,
+    //TODO: SUBMIT BUTTON CHANGE COLOR BASED ON IF PHOTO WAS SELECTED
+  };
+  
+  async componentDidMount() {
+    await Permissions.askAsync(Permissions.CAMERA_ROLL);
+  }
+
+  render() {
+    let { image } = this.state;
+
+    return (
+      <View style={styles.container}>
+
+        <View style = {styles.header}>
+          <KitText 
+          size={50} 
+          fontWeight={FontStyles.FONT_WEIGHT_BOLD} 
+          color={Colors.KIT_BLACK}>
+            Challenge Name
+        </KitText>
+
+        <KitText 
+          size={17}
+          fontWeight={FontStyles.FONT_WEIGHT_REGULAR} 
+          color={Colors.KIT_BLACK}>
+            Challenge Description
+        </KitText>
+        </View>
+
+        <View
+          style={styles.mainBody}>
+          <Image source={{ uri: image }} style={styles.imageFormat} />
+        </View>
+
+        <KitButton
+          onPress={this._pickImage}
+          style={{button: styles.photoButton}} 
+          buttonTextColor={Colors.KIT_WHITE} 
+          buttonFontWeight={FontStyles.FONT_WEIGHT_REGULAR} 
+          buttonFontSize={22} 
+          buttonTextStyle={styles.photoButtonText}
+          buttonBackgroundColor={Colors.KIT_GREEN} >
+          CHOOSE PHOTO
+        </KitButton>
+
+        <View style = {styles.submitButton}>
+
+        <KitButton
+          onPress={() => this._handleImagePicked(image)}
+          image={require("../assets/images/submitArrow.png")}
+          imageFormat={styles.submitButton}
+          style={{button: styles.submitButtonFormat}} 
+          buttonTextColor={Colors.KIT_BLACK} 
+          buttonFontWeight={FontStyles.FONT_WEIGHT_REGULAR} 
+          buttonFontSize={17} 
+          buttonTextStyle={styles.photoButtonText}
+          buttonBackgroundColor={Colors.KIT_WHITE} >
+          SUBMIT
+        </KitButton>
+        </View>
+      </View>
+    );
+  }
+
+  _pickImage = async () => {
+    let pickerResult = await ImagePicker.launchImageLibraryAsync({
+      allowsEditing: true,
```

Comment:
I'm not a 100% on what this does, but I'm assuming it's just a library implementation.

Not blocking